docs/node/tutorial.mdx

```diff
@@ -0,0 +1,151 @@
+    "rpc_addr": "tcp://127.0.0.1:26657"
+  },
+  "sync": {
+    "app_hash": "",
+    "best_block_hash": "",
+    "best_block_height": 0,
+    "best_block_time": "1969-12-31T18:00:00-06:00",
+    "syncing": false
+  },
+  "validator": {
+    "pubkey": "22cbbb666c26b2c1f42502df72c32de4d521138a1a2c96121d417a2f341a759c",
+    "power": 1
+  }
+}
+```
+
+The `node.node_id` and `node.listen_addr` fields can be given to the second node as a persistent peer, in the format: `node-id@p2p-address`.
+For the above node, we would have `b624b3395f22f8cb57cf86d58cbcd6b355d12e18@127.0.0.1:26656`.
+
+## Creating A Second Peer Node
+
+We will use the `kwil-admin` tool to create the second node's directories and configs. We can specify where it can find the genesis file,
+as well as peers that it should try to connect to when starting up. 
+
+```bash
+kwil-admin setup init --output-dir ./testnet/node1 --genesis ./testnet/node0/genesis.json \
+--peer b624b3395f22f8cb57cf86d58cbcd6b355d12e18@127.0.0.1:26656
+```
+
+To verify that the second node has been created correctly, we can start it up. Since we are running both of these locally, we need to specify
+new addresses for the different endpoints, so that the second node does not conflict with the first node:
+
+```bash
+kwild --root-dir=./testnet/node1 --app.grpc-listen-addr=localhost:50052 \
+--chain.p2p.persistent-peers=f29785e8f9c52635b55f6459b4b3a7330d3dee2e@127.0.0.1:26656 \
+--app.http-listen-addr=localhost:8081 --app.admin-listen-addr=unix:///tmp/node1.sock \
+--chain.rpc.listen-addr=tcp://127.0.0.1:26658 --chain.p2p.listen-addr=tcp://127.0.0.1:26655
+```
+
+If done correctly, you should be able to see the second node's logs, and it should be able to read the network logs.
+
+## Upgrading The Second Node To A Validator
+
+To upgrade the second node to a validator, we will use the `kwil-admin` tool to:
+1. Request to join the network as a validator, from the second node
+2. Approve the request, from the first node
+
+We will use the configured UNIX socket to communicate with the nodes.
+
+### Requesting To Upgrade
+
+To request to join from the second node:
+
+```bash
+kwil-admin validators join --rpcserver unix:///tmp/node1.sock
+```
+
+To check that the request was successful, we can get a list of pending join requests:
+
+```bash
+$ kwil-admin validators list-join-requests --rpcserver unix:///tmp/node0.sock
+Pending join requests (1 approval needed):
+  Candidate                                                       | Power | Approvals | Expiration Height
+ a2d8e307117695c8d0d3adbda136cb2ad630c43b8672a0eba46d79501831b794 |     1 |     10000 | 15906
+```
+
+### Approving The Request
+
+We can vote to approve the join request from the first node. We specify the candidate's public key, which we can get from
+the list of pending join requests as shown above:
+
+```bash
+kwil-admin validators approve a2d8e307117695c8d0d3adbda136cb2ad630c43b8672a0eba46d79501831b794 \
+--rpcserver unix:///tmp/node0.sock
+```
+
```

Comment:
```suggestion

:::tip
If the requesting node's public key is unknown, all pending requests and their public keys can be retrieved with the `kwil-admin validators list-join-requests` command.

```

Reply:
Is it necessary to have this tip?  We do it directly above

Reply:
Like literally 10 lines above